main.go

```diff
@@ -68,8 +97,28 @@ func main() {
 		r.Stop()
 	}()
 
+	go func() {
+		log.Printf("[INFO] starting HTTP endpoints ...")
+
+		mux := http.NewServeMux()
+		mux.Handle("/__/", op.NewHandler(
+			op.NewStatus("ingress53", "ingress53 updates Route53 DNS records based on the ingresses available on the kubernetes cluster it runs on").
+				AddOwner("infrastructure", "#infra").
+				AddLink("github", "https://github.com/utilitywarehouse/ingress53").
+				SetRevision(appGitHash).
+				AddChecker("running", func(cr *op.CheckResponse) { cr.Healthy("service is running") }).
```

Comment:
I see your point. I'll raise an issue and implement that in a later iteration if that's okay?